lib/parse.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+const normalize = require('./normalize');
+const normalize2 = require('./normalize-harmony');
+
+const normalizers = {
+    '1.0': normalize,
+    '2.0': normalize2,
```

Comment:
`normalize-harmony` is not **BEMDECL v2.0**.

**BEMDECL v2.0** now is not supported (#3).

I think need throw error: "The BEMDECL 2.0 format is not supported.".